vars/skipStage.groovy

```diff
@@ -157,17 +155,31 @@ boolean skip_ftest_valgrind(String distro, String target_branch) {
            target_branch =~ branchTypeRE('weekly')
 }
 
-boolean skip_ftest_hw(String size, String target_branch) {
-    return already_passed() ||
+boolean skip_ftest_hw(String size, String distro, String target_branch) {
```

Comment:
I'm not sure that `distro` needs to be an argument here and cannot just be queried within the function itself.